packages/account/src/Sections/Security/LoginHistory/login-history.jsx

```diff
@@ -17,16 +17,16 @@ const getFormattedData = login_history => {
         data[i] = {};
         const environment = login_history[i].environment;
         const environment_split = environment.split(' ');
-        const dp2p_UA = environment.match(
-            /(?<date>[0-9a-zA-Z-]+\s[0-9:]+GMT)[\s](IP=)(?<ip>[\w:.]+)\sIP_COUNTRY=(?<country>([a-zA-Z]{2}))\s(User_AGENT=)(\w.*)(?<name>iPhone|Android)([\W\w]+)\s(?<app>DP2P)(?<version>[\w\W]+)\s(LANG=)([\w]{2})/
+        const mobile_app_UA = environment.match(
+            /(?<date>[0-9a-zA-Z-]+\s[0-9:]+GMT)[\s](IP=)(?<ip>[\w:.]+)\sIP_COUNTRY=(?<country>([a-zA-Z]{2}))\s(User_AGENT=)(\w.*)(?<name>iPhone|Android)([\W\w]+)\s(?<app>Deriv P2P|Deriv GO)(?<version>[\w\W]+)\s(LANG=)([\w]{2})/
```

Comment:
updating the regex to match Deriv Go logout environment.
Sample environment 
`environment: "12-Apr-22 09:14:59GMT IP=45.120.201.159 IP_COUNTRY=AE User_AGENT=Mozilla/5.0 (Linux; U; Android 12; SM-S908E Build/SP1A.210812.016) Deriv GO/0.8.5+8 LANG=EN"`